dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataverse/DataverseSaver.java

```diff
@@ -97,4 +105,36 @@
         return Either.right(dataverse);
     }
 
+    public Dataverse saveFeaturedDataverse(Dataverse dataverse, List<Dataverse> featuredDataverses) {
+
+        return commandEngine.submit(new UpdateDataverseCommand(dataverse, null,
+                                                               featuredDataverses,
+                                                               dvRequestService.getDataverseRequest(),
+                                                               null));
+    }
+
+    /**
+     * Operation to link one dataverse to the other.
+     */
+    public Dataverse saveLinkedDataverse(long dataverseToBeLinkedId, Dataverse dataverse) {
```

Comment:
Yeah, I would probably go for taking the dataverseToBeLinked on Page for now, since the we already have dataverse from db it would be kinda waste to make another call to db.